Pear/SceneDelegate.swift

```diff
@@ -92,3 +94,47 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
     }
 
 }
+
+extension SceneDelegate: UNUserNotificationCenterDelegate {
+
+    private func setupLocalNotifications() {
+        let center = UNUserNotificationCenter.current()
+        center.requestAuthorization(options: [.alert, .sound]) { (granted, error) in
+            print("granted: \(granted)")
+        }
+        center.delegate = self
+        // get rid of previously scheduled notifications
+        center.removeAllDeliveredNotifications()
+        center.removeAllPendingNotificationRequests()
+        createNotifications(center: center, day: 2, hour: 8)
+        createNotifications(center: center, day: 4, hour: 14)
+        createNotifications(center: center, day: 6, hour: 12)
+    }
+
+    private func createNotifications(center: UNUserNotificationCenter, day: Int, hour: Int) {
+        let content = UNMutableNotificationContent()
+        content.title = "Meet your new pear!"
+        content.body = "Set up this week's chat today 😊"
+        content.sound = UNNotificationSound.default
+        var dateComponents = DateComponents()
+        dateComponents.weekday = day
+        dateComponents.hour = hour
+        dateComponents.minute = 0
+        dateComponents.second = 0
+        let trigger = UNCalendarNotificationTrigger(dateMatching: dateComponents, repeats: true)
+        let uuid = UUID().uuidString
+        let request = UNNotificationRequest(identifier: uuid, content: content, trigger: trigger)
+        center.add(request)
+    }
+
+    func userNotificationCenter(_ center: UNUserNotificationCenter, willPresent notification: UNNotification, withCompletionHandler completionHandler: @escaping (UNNotificationPresentationOptions) -> Void) {
+        completionHandler([.banner, .sound])
+    }
+
+    func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+        let navController = UINavigationController(rootViewController: FeedbackViewController())
+        navController.modalPresentationStyle = .overFullScreen
+        self.window?.rootViewController?.present(navController, animated: true, completion: nil)
```

Comment:
```suggestion
        window?.rootViewController?.present(navController, animated: true)
```

Don't say `self` if you don't need to. Also completion is nil by default.